docs: update the README page (#2)
lvgl/lv_esp_idf: README.md

## Getting started

There are currently two ways of using this repository, via the LVGL project creator, 
or configuring it as regular ESP-IDF project, for the later this documentation will
cover the command line method, however the user can set this repo using the 
[Espressif IDE](https://docs.espressif.com/projects/espressif-ide/en/latest/index.html) 

### Using the LVGL Project Creator

* Obtain the [LVGL Project Creator](https://lvgl.io/tools/project-creator)
* Launch the Project Creator
* Create an ESP IDF project 
* Follow the remaining steps given by the Project Creator
    * Set up [ESP-IDF](https://docs.espressif.com/projects/esp-idf/en/latest/esp32s3/get-started/index.html)

### Using the regular ESP-IDF project method
 
* First of all you need [ESP-IDF]((https://docs.espressif.com/projects/esp-idf/en/latest/esp32s3/get-started/index.html)

* Clone this repository on your preferred folder.
* Set the IDF environment: `$ . /path/to/esp-idf/export.sh`
* Go to this project root: `$ mkdir /path/to/this/repo`
* Update the components (using IDF set target is one option): `$ idf.py set-target esp32s3` 
* Wait the project to set-up.

> 

Reviewer: Do they have to remove/change "bsp_replaceme" from idf_component.yml or is it fine?

## Building and Flashing

After setting up this project using either the LVGL Project Creator or the Espressif tools, you
should be able to build and flash the application following the steps:

* Set the IDF environment: `$ . /path/to/esp-idf/export.sh`

> * Update the project for your target if it has did not: `$ idf.py set-target <esp32xxx>`

Reviewer: ```suggestion
* Update the project for your target if it is not already: `$ idf.py set-target <esp32xxx>`
```

# LVGL under ESP-IDF demonstration project

## Overview

The *lv_esp_idf* presents a common way of building LVGL applications on top 
of the ESP-IDF structure. This project is intended to serve as a references
for users that just wanted to dive in the application code while they
abstract the project set-up complexities.

The main application will set-up the LVGL, plus the display and touch driver(if any),
then it will execute one of the selected LVGL demos such as benchmarking or music demo.

## Supported Boards

As any ESP-IDF project, this one can support a variety of ESP32 families boards, the user can
check the out-of-the-box supported boards by referring the folder manifests,

## Getting started

> There are currently two ways of using this repository, via the LVGL project creator, 

Author: it is already mentioned here: https://github.com/lvgl/lv_esp_idf/blob/64193caae4dfe94f109ad0eacd7d60f077911dda/README.md?plain=1#L27 as part of the step by step guide.
